elections/2020/governance-committee-election.md

```diff
@@ -164,15 +164,305 @@ election).
+* [romil-punetha](https://github.com/romil-punetha) (no email found)
+* [romilpunetha](https://github.com/romilpunetha) (no email found)
+* [rubenvp8510](https://github.com/rubenvp8510)
+* [safris](https://github.com/safris)
+* [samvaity](https://github.com/samvaity)
+* [satac2](https://github.com/satac2) (no email found)
+* [saturnism](https://github.com/saturnism)
+* [seh](https://github.com/seh)
+* [sfriberg](https://github.com/sfriberg) (no email found)
+* [shaochengwang](https://github.com/shaochengwang) (no email found)
+* [shengxil](https://github.com/shengxil) (no email found)
+* [shirou](https://github.com/shirou)
+* [shivkanya9146](https://github.com/shivkanya9146)
+* [shovnik](https://github.com/shovnik)
+* [siyopao](https://github.com/siyopao)
+* [sjbrunst](https://github.com/sjbrunst)
+* [sleighzy](https://github.com/sleighzy)
+* [snehilchopra](https://github.com/snehilchopra) (no email found)
+* [sonofachamp](https://github.com/sonofachamp) (no email found)
+* [sribalakumar](https://github.com/sribalakumar) (no email found)
+* [srjames90](https://github.com/srjames90)
+* [stefanprisca](https://github.com/stefanprisca)
+* [svrnm](https://github.com/svrnm)
+* [tarekgh](https://github.com/tarekgh)
+* [tedpennings](https://github.com/tedpennings)
+* [tedsuo](https://github.com/tedsuo)
+* [tensorchen](https://github.com/tensorchen)
+* [thisthat](https://github.com/thisthat)
+* [tigrannajaryan](https://github.com/tigrannajaryan)
+* [toumorokoshi](https://github.com/toumorokoshi)
+* [trask](https://github.com/trask)
+* [tsloughter](https://github.com/tsloughter)
+* [tylerbenson](https://github.com/tylerbenson)
+* [udekel](https://github.com/udekel) (no email found)
+* [vikrambe](https://github.com/vikrambe)
+* [vmarchaud](https://github.com/vmarchaud)
+* [vmihailenco](https://github.com/vmihailenco)
+* [wangzlei](https://github.com/wangzlei) (no email found)
+* [wilguo](https://github.com/wilguo) (no email found)
+* [williamhu99](https://github.com/williamhu99) (no email found)
+* [wingyplus](https://github.com/wingyplus)
+* [wtong98](https://github.com/wtong98)
+* [wtyanan](https://github.com/wtyanan)
+* [xiang17](https://github.com/xiang17)
+* [xiao-lix](https://github.com/xiao-lix)
+* [xukaren](https://github.com/xukaren)
+* [yatender-oktalk](https://github.com/yatender-oktalk)
+* [ymotongpoo](https://github.com/ymotongpoo) (no email found)
+* [youngbupark](https://github.com/youngbupark)
+* [yue9944882](https://github.com/yue9944882)
+* [yurishkuro](https://github.com/yurishkuro)
+* [yyyogev](https://github.com/yyyogev)
+* [z1c0](https://github.com/z1c0)
+* [zeitlinger](https://github.com/zeitlinger)
+* [zenmoto](https://github.com/zenmoto)
+* [ziqizh](https://github.com/ziqizh) (no email found)
+
+We will contact all of the above via a GitHub issue and via email (where possible) to register themselves to vote in the election via this Google form: [https://forms.gle/2VWUNA8oz9DP4C3SA](https://forms.gle/2VWUNA8oz9DP4C3SA)
```

Comment:
> "to register themselves to vote in the election"

may be read as actual election process. Perhaps expand a little bit saying that actual election will be made via CIVS.

Reply:
I tried to clarify, thanks.